src/modules/hide_sidebar_elements/index.js

```diff
@@ -48,7 +48,7 @@ class HideSidebarElementsModule {
         if (settings.get('hideFeaturedChannels')) {
             if (removeFeaturedChannelsListener) return;
 
-            removeFeaturedChannelsListener = domObserver.on('.side-nav-section a[data-test-selector="recommended-channel"], .side-nav-section a[data-test-selector="similarity-channel"], .side-nav-section .tw-svg__asset--navchannels', (node, isConnected) => {
+            removeFeaturedChannelsListener = domObserver.on('.side-nav-section a[data-test-selector="recommended-channel"], .side-nav-section a[data-test-selector="similarity-channel"], .side-nav-section .tw-svg__asset--navchannels, div[aria-label="Recommended Channels"]', (node, isConnected) => {
```

Comment:
how does this handle i18n?

Reply:
> how does this handle i18n?

right... 🤦 if they end up removing data-test-selectors, targetable classes etc. Maybe one possibility would be to search for react props/keys and identify nodes to hide that way.